### ISSUE TITLE
:set colorcolumn[...] results in illegible text

### ISSUE THREAD

**Issue by zatchheems** [Rigellute/rigel#2]
Text is illegible when `colorcolumn` is set. This is easily remedied by setting `guifg` and `guibg` to `NONE` and replacing `guibg` and `ctermbg` with dark blues like so:

```vim
hi ColorColumn guifg=NONE ctermfg=NONE guibg=#002230 ctermbg=230 gui=NONE cterm=NONE
```

#### With `:set colorcolumn=100`
![rigel-colorcolumn-bug](https://user-images.githubusercontent.com/8506829/63800953-dbeee800-c8d4-11e9-8f9f-7c2c3a2a44bb.png)

#### With `:set colorcolumn=100` and highlight edits
![rigel-colorcolumn-fixed](https://user-images.githubusercontent.com/8506829/63800954-dbeee800-c8d4-11e9-8b35-20469c3be6b3.png)

The dark blue for the colorcolumn might be a bit too subtle, but the basic idea is there. Could possibly make it the same color as the cursorline (`guibg=#001a25 ctermbg=234`), like so:

![image](https://user-images.githubusercontent.com/8506829/63804127-127c3100-c8dc-11e9-9237-68376ac257d4.png)

**Comment by Rigellute** [comment]
Thanks for this - one major motivation for creating this theme was for better support for JSX and Typescript/Flow types.

And you're right, I had assigned the wrong colours for these properties.

I've fixed in https://github.com/Rigellute/rigel/commit/4d96cbb1af73a8d1d0a562c0f71ec7cfe391b09b

Let me know if this works for you!

<img width="419" alt="image" src="https://user-images.githubusercontent.com/12150276/64082873-0e9e4380-cd0e-11e9-9361-8f75d08e9c58.png">
